tags

```diff
@@ -0,0 +1,70 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
What's the significance of this file? Can probably be removed.

Reply:
Yes, it's a file created by a program installed on my machine. I just removed it.